<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qsc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qsc.qrc">
    <normaloff>:/images/images/qsc.png</normaloff>:/images/images/qsc.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCliente">
    <property name="title">
     <string>Cliente</string>
    </property>
    <addaction name="actionAdicionar"/>
    <addaction name="actionListar"/>
   </widget>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuA_es">
    <property name="title">
     <string>Compra</string>
    </property>
    <addaction name="actionComprar"/>
    <addaction name="actionPagar"/>
   </widget>
   <addaction name="menuA_es"/>
   <addaction name="menuCliente"/>
   <addaction name="menuOp_es"/>
  </widget>
  <action name="actionComprar">
   <property name="icon">
    <iconset resource="qsc.qrc">
     <normaloff>:/images/images/Shopping-Cart-Add.png</normaloff>:/images/images/Shopping-Cart-Add.png</iconset>
   </property>
   <property name="text">
    <string>Comprar</string>
   </property>
  </action>
  <action name="actionAdicionar">
   <property name="icon">
    <iconset resource="qsc.qrc">
     <normaloff>:/images/images/User-add.png</normaloff>:/images/images/User-add.png</iconset>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="qsc.qrc">
     <normaloff>:/images/images/Tools.png</normaloff>:/images/images/Tools.png</iconset>
   </property>
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionPagar">
   <property name="icon">
    <iconset resource="qsc.qrc">
     <normaloff>:/images/images/Credit-card-Visa.png</normaloff>:/images/images/Credit-card-Visa.png</iconset>
   </property>
   <property name="text">
    <string>Pagar</string>
   </property>
  </action>
  <action name="actionListar">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qsc.qrc"/>
 </resources>
 <connections/>
</ui>
